minard/channeldb.py

```diff
@@ -416,3 +416,11 @@ def upload_channel_status(form):
         "%(disable_n100)s, %(disable_n20)s, %(high_dropout)s, "
         "%(bad_base_current)s, %(bad_data)s, %(name)s, %(reason)s, %(info)s)",
          form.data)
+
+def get_current_run():
+
+    conn = engine.connect()
+
+    result = conn.execute("SELECT last_value FROM run_number")
+
+    return result.fetchone()[0]
```

Comment:
This already exists in the detector_state module, so you should be able to use that directly in views.py.